### ISSUE TITLE
Error when no variants on VEP stats file

### ISSUE THREAD

**Issue by fgvieira** [ewels/MultiQC#1446]
**Description of bug:**

MultiQC gives an error when no variants are present in the VEP stats file.

**MultiQC Error log:**

```
╭────────────────────────────────────────────────────────────── Oops! The 'vep' MultiQC module broke... ───────────────────────────────────────────────────────────────╮
│ Please copy this log and report it at https://github.com/ewels/MultiQC/issues                                                                                        │
│ Please attach a file that triggers the error. The last file found was: stats/variants/delly/real.annot_vep.stats                                                     │
│                                                                                                                                                                      │
│ Traceback (most recent call last):                                                                                                                                   │
│   File "lib/python3.9/site-packages/multiqc/multiqc.py", line 604, in run                   │
│     output = mod()                                                                                                                                                   │
│   File "lib/python3.9/site-packages/multiqc/modules/vep/vep.py", line 40, in __init__       │
│     self.parse_vep_txt(f)                                                                                                                                            │
│   File "lib/python3.9/site-packages/multiqc/modules/vep/vep.py", line 151, in parse_vep_txt │
│     existing = values[1].split("(")[0].replace(" ", "")                                                                                                              │
│ IndexError: list index out of range                                                                                                                                  │
╰──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯

```

**File that triggers the error:**

The issues occurs when the file has no SNVs (only SVs), and VEP sets line `Novel / existing variants` to `-`:

```
[...]
[General statistics]
Lines of input read     6
Variants processed      6
Variants filtered out   0
Overlapped genes        49
Overlapped transcripts  49
Overlapped regulatory features  0
Novel / existing variants       -

[Variant classes]
deletion        6

[Consequences (most severe)]
transcript_ablation     1
frameshift_variant      3
upstream_gene_variant   2

[...]
```

**MultiQC run details (please complete the following):**

- Command used to run MultiQC: `multiqc .`
- MultiQC Version: `MultiQC v1.1dev0`
- Operating System: `Ubuntu 20.04`
- Python Version: `Python 3.9`
- Method of MultiQC installation: `conda`

**Comment by ewels** [comment]
Thanks @fgvieira - I'll take a look.

Is it possible to attach the full log as a file instead of a snippet please? Then I can run MultiQC on it myself to replicate the issue and be sure that I've fixed it.

**Comment by fgvieira** [comment]
The full log or the full file that triggers the error?

**Comment by ewels** [comment]
The file that triggers the error - a minimal test case that I can run locally to reproduce the error. But a full file, as there are often secondary errors when fixing these ZeroDivisionError bugs..

**Comment by fgvieira** [comment]
File that triggers error:

[delly.annot_vep.txt](https://github.com/ewels/MultiQC/files/6647269/delly.annot_vep.txt)

log:

[multiqc_variants_annot.log](https://github.com/ewels/MultiQC/files/6647292/multiqc_variants_annot.log)

**Comment by ewels** [comment]
Many thanks @fgvieira!

@ErikDanielsson has just fixed this for you in #1450 - now available in version 1.11dev. Let us know if you hit any problems with it 👍🏻 

Phil
